packages/forma-36-fcss/scss/styles.scss

```diff
@@ -1,4 +1,4 @@
-@import '../node_modules/@contentful/forma-36-tokens/dist/scss/index.scss';
+@import '../../../node_modules/@contentful/forma-36-tokens/dist/scss/index.scss';
```

Comment:
Because yarn workspaces moves our dependencies to the root I had to do this

I tried to do something like `@contentful/forma-36-tokens/dist/scss/index.scss`, but for this to work something is still missing
we could do it in another PR I think

Reply:
Hmm, worried this change will break things when using the package in another project

Reply:
I think you're right @m10l, but this was brittle before anyway because of the relative path. We should find out how to get the absolute import working.

Reply:
I will try more!
I suspect that we need to tell webpack or whatever is building the packages that `@contentful/forma-36-tokens` is a real dependency in node_modules in the root

And again, my change is just updating the relative path. The version in production is using a relative path already

Reply:
Hey! I managed to make the relative path work, but I needed to add this dependency
https://www.npmjs.com/package/node-sass-import

And to ask yarn to not hoist it because it's not working properly when hoisted

And then we use it in the npm-script `build` like this:
`node-sass scss/styles.scss -o dist --output-style compressed --importer node_modules/node-sass-import`